roles/kubernetes-apps/ansible/tasks/main.yml

```diff
@@ -29,6 +30,14 @@
   tags:
     - coredns
 
+- name: Kubernetes Apps | nodelocalDNS
+  import_tasks: "tasks/nodelocaldns.yml"
+  when:
+    - enable_nodelocaldns == True
+    - inventory_hostname == groups['kube-master'][0]
+  tags:
+    - coredns
```

Comment:
I think this needs to be `nodelocaldns`?

Reply:
correct! thanks for the catch